website/src/models/challenge.js

```diff
@@ -97,7 +97,6 @@ schema.methods.syncToUser = async function syncChallengeToUser (user) {
   let [challengeTasks, userTasks] = await Q.all([
     // Find original challenge tasks
     Tasks.Task.find({
-      userId: {$exists: false},
```

Comment:
Oh... eh, I get it now. I'm sorry for misunderstanding. I'll add it make and correct the tests as I should have before.